maintenance/health/servicehealthcheck/README.md

```diff
@@ -0,0 +1,5 @@
+# Health Checks for Services 
+* makes it possible to have a health check for different services (e.g. postgres, redis)
+* A service has to implement the HealthCheck interface and register the healthCheck with `healthchecker.RegisterHealthCheck(hc HealthCheck)`
+* `healthchecker` does NOT make any caching of the results but it offers a struct (`ConnectionState`) for caching
+ the result in the Implementations of the Health Checks
```

Comment:
Please straighten this readme. Like indentions, usage of backticks (`), capitalised letters. Also "does NOT cache the results".